visualization/app/codeCharta/core/multipleFile/multipleFile.service.ts

```diff
@@ -91,6 +91,7 @@ export class MultipleFileService {
 		let outputMap: CodeMap = {
 			projectName: "Aggregation of following projects: " + this.projectNameArray.join(", "),
 			fileName: "Aggregation of following files: " + this.fileNameArray.join(", "),
+			apiVersion: "1.0",
```

Comment:
I think this is a bit more complex, as we should definitely check the versions of both aggregated maps. What should happen if they have different api versions?

Reply:
That is a valid question. If we allow to import maps with different apiVersions at the same time, what is the expected apiVersion of the aggregate, if downloaded? Are we planning to update the maps to the newer version and will we even allow to download an aggregate in this case?

Since these questions might not be trivial to answer, I just implemented this - admitedly simplistic - fix to make sure the download is a valid cc.json file, that can be imported to codecharta again.
But if we have answers to the above questions I agree implementing a target solution is better than this fix.

Reply:
if we can aggregate the maps without errors, we should allow the option to download it. I think this means: 
- Selected maps might have different apiVersions (we might want to handle that in the future), 
- but (I guess) we always create a new valid `codeMap`/cc.json, which is using the latest apiVersion. 
- The question is just: where to get the latest apiVersion from.. [Here](https://github.com/MaibornWolff/codecharta/commit/c274ce1e3f4103ac94874fed58bbd26ecac0f52a) you see, where the apiVersion is persisted.

Reply:
yes i agree, maps that can be aggregated should be downloadable.

so, if i understand correctly, the target solution is to take the current codeCharta version number as the apiVersion (regardless of the apiVersion of the aggregated maps), since this will always lead to a valid .cc.json. That sounds reasonable, i will implement it like this.

Reply:
yes, that's what I thought, because we will create a new map (if possible) nevertheless. And if thats possible, it gets the current apiVersion